internal/cluster/refresh_status_preprocessor.go

```diff
@@ -53,16 +54,27 @@ func (r *refreshPreprocessor) preprocess(ctx context.Context, c *clusterPreproce
 	checkValidationsInStatuses := []string{
 		models.ClusterStatusInsufficient, models.ClusterStatusReady, models.ClusterStatusPendingForInput, models.ClusterStatusPreparingForInstallation,
 	}
+	var ignoredValidations []string
+	var err error
+	if c.cluster != nil {
+		ignoredValidations, err = common.GetIgnoredValidations(c.cluster.IgnoredClusterValidations, "cluster", string(*c.cluster.ID))
```

Comment:
maybe just to provide clustet? and type?

Reply:
I think actually providing c.cluster.IgnoredClusterValidations should be enough, just print error here. We will get everything we need from this log line